src/language-js/print/member-chain.js

```diff
@@ -102,28 +103,32 @@ function printMemberChain(path, options, print) {
           shouldInsertEmptyLineAfter(node) ? hardline : "",
         ],
       });
-      path.call((callee) => rec(callee), "callee");
     } else if (isMemberish(node)) {
-      printedNodes.unshift({
+      path.call((object) => rec(object), "object");
+      printedNodes.push({
```

Comment:
I reversed the order:
- before: `unshift()` current property -> recursive call for object
- after: recursive call for object -> `push()` current property

This is because I need the doc for the object to print member lookup.
I think the performance and readability might be worried about. My thoughts are following, though I haven't benchmarked.
- Performance
  - Tail call optimization doesn't apply for both before and after.
  - `push()` should be faster than `unshift()`
- Readability
  - Former one goes right to left and latter one goes left to right. I feel it get more readable.

Reply:
I reverted this change because objectDoc doesn't look necessary here.